src/librustc_resolve/diagnostics.rs

```diff
@@ -1674,6 +1674,29 @@ fn main() {
 ```
 "##,
 
+E0910: r##"
+You tried to use something as a trait bound that is not a trait.
```

Comment:
You tried to use something that is not a trait as a trait bound.

Reply:
Fixed